ivy/functional/frontends/paddle/tensor/manipulation.py

```diff
@@ -173,3 +173,14 @@ def take_along_axis(arr, indices, axis):
 @to_ivy_arrays_and_back
 def rot90(x, k=1, axes=(0, 1), name=None):
     return ivy.rot90(x, k=k, axes=axes)
+
+
+@with_supported_device_and_dtypes(
```

Comment:
This should be `with_supported_dtypes` if there are no device specifications

Reply:
resolved @bipinKrishnan . Thanks